cmd/kubeadm/app/cmd/init.go

```diff
@@ -120,6 +121,7 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 		Short: "Run this command in order to set up the Kubernetes master.",
 		Run: func(cmd *cobra.Command, args []string) {
 			var err error
+
```

Comment:
nit: can you remove these extra lines? doesn't seem related to changeset. prefer to keep diffs small :)

Reply:
Sure. This is not a final push. This is just to show approach and get confirmation from members but thanks, I'll keep that in mind  :)